src/Controller/Api/ApiLoginController.php

```diff
@@ -21,6 +22,8 @@ class ApiLoginController implements LoggerAwareInterface
 
     private Security $security;
 
+    private LoggerInterface $analyticsLogger;
```

Comment:
you brought this in the class, you don't inject it and you don't use it :smile: something's fishy here